pkg/appmanager/appManager.go

```diff
@@ -1867,47 +1866,45 @@ func (appMgr *Manager) recordIngressEvent(ing *v1beta1.Ingress,
 func getEndpointsForService(
 	portName string,
 	eps *v1.Endpoints,
-) []string {
-	var ipPorts []string
+) []Member {
+	var members []Member
 
 	if eps == nil {
-		return ipPorts
+		return members
 	}
 
 	for _, subset := range eps.Subsets {
 		for _, p := range subset.Ports {
 			if portName == p.Name {
-				port := strconv.Itoa(int(p.Port))
 				for _, addr := range subset.Addresses {
-					var b bytes.Buffer
-					b.WriteString(addr.IP)
-					b.WriteRune(':')
-					b.WriteString(port)
-					ipPorts = append(ipPorts, b.String())
+					member := Member{
+						Address:    addr.IP,
+						Port:       p.Port,
+						Session:   "user-enabled",
+					}
+					members = append(members, member)
 				}
 			}
 		}
 	}
-	if 0 != len(ipPorts) {
-		sort.Strings(ipPorts)
-	}
-	return ipPorts
+	return members
```

Comment:
We aren't sorting anymore - is this something we are counting on CCCL to do for us?

Reply:
It doesn't matter for CCCL, was there a reason for sorting them?

Reply:
I'm not sure why it was done here before (I agree it doesn't matter to CCCL) but I saw it was removed. We aren't doing it in the NodePort case, so I think it's fine to remove the sorting here.

Reply:
Removing the sorting was intentional, it does not matter for CCCL.